content/en/agent/guide/agent-fips-proxy.md

```diff
@@ -0,0 +1,325 @@
+---
+title: Datadog FIPS Compliance
+kind: Guide
+disable_toc: false
+further_reading:
+- link: "agent/proxy"
+  tag: "Documentation"
+  text: "Agent Proxy Configuration"
+algolia:
+  rank: 80
+  tags: ["fips", "fips proxy", "compliance"]
+---
+
+{{< site-region region="us,us3,us5,eu,ap1" >}}
+<div class="alert alert-warning">The Datadog FIPS proxy is only available in the US1-FED region.</a></div>
+{{< /site-region >}}
+
+The Datadog FIPS proxy ensures that communication between the Datadog Agent and Datadog uses FIPS-compliant encryption.
+
+The Datadog FIPS proxy is a separately-distributed component that is deployed on the same host as the Datadog Agent. The proxy acts as an intermediary between the Agent and Datadog. The Agent communicates with the Datadog FIPS proxy, which encrypts payloads using a FIPS 140-2 validated cryptographic module and relays the payloads to Datadog.
+
+## Supported platforms and limitations
+
+The Datadog FIPS proxy's compliance is based on its use of the FIPS 140-2 validated [Cryptographic Module - Certificate #4024][1]. See the related [security policy][2] for information around validated operating environments and restrictions. It is ultimately the end user's responsibility to ensure operating environment compliance with the security policy and wider FIPS guidance.
+
+Supported platforms (64-bit x86 only):
+- RHEL >= 7
+- Debian >= 8
+- Ubuntu >= 14.04
+
+**Note**: The Datadog FIPS proxy does not support Cluster Agent and Serverless Monitoring.
```

Comment:
We'll need to bring more details about the Cluster Agent since EKS/ECS were added

Reply:
+1. The cluster-agent is supported to a degree now for outbound communication (cluster-agent -> Datadog) but not the CLC runners to cluster-agent.

Reply:
I've added this to the latest commit